empress/reconcile/recon_vis/utils.py

```diff
@@ -172,31 +179,43 @@ def dict_to_recongraph(old_recon_graph: Dict[Tuple, List]):
 
 # Temporal ordering utilities
 
-def build_trees_with_temporal_order(host_tree: dict, parasite_tree: dict, reconciliation: dict) \
-        -> Tuple[tree.Tree, tree.Tree, bool]:
+def build_trees_with_temporal_order(host_tree: dict, parasite_tree: dict, reconciliation: dict,) \
```

Comment:
Is there a reason you added the extra `,`? It wouldn't give you syntax error but I just wonder whether it's a typo.